recipe/meta.yaml

```
@@ -213,28 +217,30 @@ outputs:
         - $RPATH/ld64.so.2  # [ppc64le]
     requirements:
       build:
-        - {{ compiler('m2w64_c') }}    # [win]
-        - {{ compiler('m2w64_cxx') }}  # [win]
-        - {{ compiler('c') }}
-        - {{ compiler('cxx') }}
-        - {{ stdlib('c') }}
-        - llvm-openmp                  # [osx]
-        - libgomp                      # [linux]
+        - {{ compiler('c') }}              # [unix]
+        - {{ compiler('cxx') }}            # [unix]
+        - {{ compiler('m2w64_c') }}        # [win]
+        - {{ compiler('m2w64_cxx') }}      # [win]
+        - {{ stdlib('c') }}                # [unix]
+        - {{ stdlib('m2w64_c') }}          # [win]
+        - llvm-openmp                      # [osx]
+        - libgomp                          # [linux or win]
+        - m2-bash                          # [win]
         - git
-        - make                         # [not win]
-        - posix                        # [win]
+        - make
         - cmake
         - ninja
-        - cross-r-base {{ r_base }}    # [build_platform != target_platform]
-        - r-base                       # [build_platform != target_platform]
-        - r-matrix                     # [build_platform != target_platform]
-        - r-data.table                 # [build_platform != target_platform]
-        - r-magrittr                   # [build_platform != target_platform]
-        - r-jsonlite                   # [build_platform != target_platform]
-        - r-knitr                      # [build_platform != target_platform]
+        - cross-r-base {{ r_base }}        # [build_platform != target_platform]
+        - r-base                           # [build_platform != target_platform]
+        - r-matrix                         # [build_platform != target_platform]
+        - r-data.table                     # [build_platform != target_platform]
+        - r-magrittr                       # [build_platform != target_platform]
+        - r-jsonlite                       # [build_platform != target_platform]
+        - r-knitr                          # [build_platform != target_platform]
       host:
         - {{ pin_subpackage('_r-xgboost-mutex', exact=True) }}
         - {{ pin_subpackage('libxgboost', max_pin='x.x.x') }}
+        - libgomp      # [linux or win]
```

Tweak selector depth to make this easier to pick out

```suggestion
        - libgomp                                                  # [linux or win]
```